docusaurus.config.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
   url: "https://docs.camunda.io",
   // baseUrl: "/camunda-cloud-documentation/",
   baseUrl: "/",
-  onBrokenLinks: "throw",
+  onBrokenLinks: "warn",
```

Comment:
Just noticing this since it's different from the other PRs. Do we want to keep this config like this in the PR?

Reply:
Do we want this on @pepopowitz? I know it was being weird on build.

Reply:
We turned this on on-purpose! I can't remember why. There was a conversation about some pages that were failing the build (pre-link-checking), and we had intended to clean them up later.

I don't see that conversation in this PR -- does anyone remember it, and know where it is? Or, @christinaausley do you know if the pages in question were moved to the correct place, and we can now remove this? 

I'll also try running a build locally with it set back to "error" and see if this one still runs fine, in which case I'd say YES to setting back to "error".

Reply:
This was in [Slack](https://camunda.slack.com/archives/C026U8GBNSW/p1680893718374489?thread_ts=1680867949.253929&cid=C026U8GBNSW)!

Reply:
I think we want to revert this to "throw".

Reply:
As in throw the error.

Reply:
It runs locally with no issue when reverted to `throw`, so I think whatever was originally inspired us to change this has since been resolved.

Reply:
Ah, @pepopowitz we may have to keep as-is tentatively as this failed.